test/end-to-end/pages/equalityanddiversity/completedpage.js

```diff
@@ -1,10 +1,14 @@
 'use strict';
 
-const commonContent = require('app/resources/en/translation/common');
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
 
-module.exports = async function() {
+module.exports = async function(language ='en') {
     const I = this;
+    const commonContent = language === 'en' ? commonContentEn : commonContentCy;
     await I.wait(3);
-    await I.waitForText(commonContent.saveAndContinue);
+    if (language ==='en') {
+        await I.waitForText(commonContent.saveAndContinue);
+    }
```

Comment:
again, not sure why you'd only want to check the text here on English journeys? It will likely help with test stability for both languages